setup.py

```diff
@@ -157,4 +159,5 @@ def _get_version():
         "Topic :: Text Processing",
     ],
     keywords="podium nlp natural-language-processing machine learning",
+    zip_safe=False,
```

Comment:
Any reason why the lib is not zip-safe?

Reply:
The 2nd note in the [linked docs](https://mypy.readthedocs.io/en/stable/installed_packages.html#creating-pep-561-compatible-packages) explains this.